cloudnet/src/main/java/de/dytanic/cloudnet/network/listener/PacketServerDatabaseAction.java

```diff
@@ -0,0 +1,154 @@
+package de.dytanic.cloudnet.network.listener;
+
+import de.dytanic.cloudnet.CloudNet;
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+import de.dytanic.cloudnet.database.AbstractDatabaseProvider;
+import de.dytanic.cloudnet.database.IDatabase;
+import de.dytanic.cloudnet.driver.network.INetworkChannel;
+import de.dytanic.cloudnet.driver.network.def.PacketConstants;
+import de.dytanic.cloudnet.driver.network.protocol.IPacket;
+import de.dytanic.cloudnet.driver.network.protocol.IPacketListener;
+import de.dytanic.cloudnet.driver.network.protocol.Packet;
+
+import java.util.UUID;
+
+public class PacketServerDatabaseAction implements IPacketListener {
+    @Override
+    public void handle(INetworkChannel channel, IPacket packet) throws Exception {
+        if (packet.getHeader().contains("message")) {
+            AbstractDatabaseProvider databaseProvider = CloudNet.getInstance().getDatabaseProvider();
+            String message = packet.getHeader().getString("message");
+
+            if (packet.getHeader().contains("database")) {
+                String databaseName = packet.getHeader().getString("database");
+
+                IDatabase database = databaseProvider.getDatabase(databaseName);
```

Comment:
You might want to check if the database is null

Reply:
> You might want to check if the database is null

The database should never be null

Reply:
Ah, yes, you're right